MiniApp/Classes/Display/MiniAppWebView.swift

```diff
@@ -12,21 +12,31 @@ internal class MiniAppWebView: WKWebView {
         return config
     }
 
-    convenience init(miniAppId: String, versionId: String) {
+    convenience init(miniAppId: String, versionId: String, queryParams: String? = nil) {
         let schemeName = Constants.miniAppSchemePrefix + miniAppId
-        let urlRequest = URLRequest(url: URL(string: schemeName + "://miniapp/" + Constants.rootFileName)!)
+        let urlRequest = URLRequest(url: URL(string: schemeName + "://miniapp/" + Constants.rootFileName + MiniAppWebView.getQueryParams(queryParams: queryParams))!)
         let config = MiniAppWebView.defaultConfig()
         config.setURLSchemeHandler(URLSchemeHandler(versionId: versionId), forURLScheme: schemeName)
         self.init(frame: .zero, configuration: config)
         commonInit(urlRequest: urlRequest)
     }
 
-    convenience init(miniAppURL: URL) {
+    convenience init(miniAppURL: URL, queryParams: String? = nil) {
         let urlRequest = URLRequest(url: miniAppURL)
         self.init(frame: .zero, configuration: MiniAppWebView.defaultConfig())
         commonInit(urlRequest: urlRequest)
     }
 
+    private static func getQueryParams(queryParams: String?) -> String {
+        guard let queryString = queryParams else {
+            return ""
+        }
+        if queryString.hasPrefix("?") {
```

Comment:
In Android side, we haven't this kind of checking, or, any other additional checking for now. 
HostApp can pass `?demoParms` or, `??demoParms` or even, `???demoParms`. So, maybe we can just return it by appending at-least one qus-mark. Therefore, `return "?" + queryString` is enough.

And, let MiniApp developers to parse in their side.

Reply:
I don't think query parameters would have multiple question marks `?`. 
Check https://tools.ietf.org/html/rfc3986#section-3.4

@corycaywood , whats your thoughts on this ?

Reply:
Done